mobify/progressive-web-scaffold
# Refactor pdp

Tweak PDP to improve its placeholder skeletons.

| Before | After |
| --- | --- |
| ![skeletons-before](https://cloud.githubusercontent.com/assets/734535/20637322/60aa1fce-b339-11e6-81cd-1b69e3d948f3.gif) | ![skeletons-after](https://cloud.githubusercontent.com/assets/734535/20637324/681a2182-b339-11e6-9cf0-42ce2b358c5a.gif) |

> Notice that in the **Before** animation that the product description is not represented in the skeleton, that the carousel jumps in size after it loads, and that the form fields are missing.

## Changes
- Update SDK to 0.10.6 and add Ratio component base styles
- Add `display` utility class for `display: block`
- Tweak `disabled` styles for buttons
- Replace remnant `c-` with `pw-` in carousel theme styles
- Tweak PDP JSX files to improve its skeleton placeholders

## How to test-drive this PR
- Run `npm run dev`
- [Preview Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit any PDP
- Confirm that the PDP skeleton placeholders look decent
